### PR TITLE
chore(Portal): allow TypeScript in Examples.tsx files

### PR DESCRIPTION
**Pull request by tujoworker**
The code changes of `react-live` to still support SSR are med [here](https://github.com/tujoworker/react-live/pull/1).

### CODE REVIEW

File: packages/dnb-eufemia/src/components/form-status/FormStatus.d.ts [hunk 1]
```diff
@@ -116,10 +116,6 @@ export interface InfoIconProps {
    * The `title` attribute in the status.
    */
   title?: string;
-
-  /**
-   * Theme name. If provided, will ensure correct theming of icon
-   */
```
Comment (langz) [inline packages/dnb-eufemia/src/components/form-status/FormStatus.d.ts:122]: Was this change intentional to have as part of this PR?

File: packages/dnb-eufemia/src/components/progress-indicator/style/themes/dnb-progress-indicator-theme-ui.scss [hunk 2]
```diff
@@ -3,8 +3,6 @@
 *
 */
 
-@import '../../../../style/core/utilities.scss';
```
Comment (langz) [inline packages/dnb-eufemia/src/components/progress-indicator/style/themes/dnb-progress-indicator-theme-ui.scss:6]: Was this change intentional to have as part of this PR?

File: packages/dnb-eufemia/src/components/progress-indicator/__tests__/__snapshots__/ProgressIndicator.test.tsx.snap [hunk 3]
```diff
@@ -281,9 +281,6 @@ exports[`ProgressIndicator scss have to match default theme snapshot 1`] = `
 * ProgressIndicator theme
 *
 */
-/*
- * Utilities
- */
```
Comment (langz) [inline packages/dnb-eufemia/src/components/progress-indicator/__tests__/__snapshots__/ProgressIndicator.test.tsx.snap:286]: Was this change intentional to have as part of this PR?

**Review by langz — APPROVED** [review]
Just gave the code/live editor a quick test, and it seems to work great.

The tabbing feature that we implemented in https://github.com/dnbexperience/eufemia/pull/2222 doesn't seem to work anymore, not that it's important though.

File: packages/dnb-eufemia/src/components/progress-indicator/__tests__/__snapshots__/ProgressIndicator.test.tsx.snap (hunk 3 above)
Comment (tujoworker) [reply packages/dnb-eufemia/src/components/progress-indicator/__tests__/__snapshots__/ProgressIndicator.test.tsx.snap:286]: nope 👌

**Comment by tujoworker** [comment]
Added a e2e test ✅
